numpy/__init__.py

```diff
@@ -157,11 +157,8 @@
     # Make these accessible from numpy name-space
     # but not imported in from numpy import *
     # TODO[gh-6103]: Deprecate these
-    if sys.version_info[0] >= 3:
-        from builtins import bool, int, float, complex, object, str
-        unicode = str
-    else:
-        from __builtin__ import bool, int, float, complex, object, unicode, str
+    from builtins import bool, int, float, complex, object, str
+    unicode = str
```

Comment:
`unicode` is also defined in `np.compat`. We may want to deal with that at some point. It is unfortunate that it is part of the default numpy api.